### ISSUE TITLE
Destruction Warlock

### ISSUE THREAD

**Issue by sMteX** [MartijnHols/WoWAnalyzer#259]
Please use this issue to discuss any suggestions or features for this spec. Let us know if you're doing something or plan on adding some feature. 

Currently maintained by @sMteX (@Chizu on WoWAnalyzer Discord).

**Comment by sMteX** [comment]
Currently open issues, TODO list:
- [ ] Immolate dot uptime
- [ ] Wasted Conflagrate / Dimensional Rift charges?
- [ ] Talent modules (damage contributed etc.)
- [ ] Notable trait effects
- [ ] Legendaries
- [ ] Set bonuses (T20 4p probably unlikely to be accurate)

**Comment by kaedys** [comment]
The analyzer reports that Summon Infernal was not used during the fight, even if one is speced into Grimoire of Supremacy.  While it's technically feasible to use Lord of Flames with Grimoire of Supremacy (by using the Internal's Meteor Strike), it costs 2 Soul Shards and 2 demon summon casts, significantly reducing the value of that artifact trait (to the point where it's generally not considered worthwhile to use).

**Comment by superderpicus** [comment]
its a dps gain to start with infernal use meteor strike, and summon doomguard. its only 1 soul shard since you start fight with infernal up already. also the movement is for general use. if you are using lessons you will know to save your rifts.

**Comment by superderpicus** [comment]
its not worth it to take supremacy usually. i will look into looking for meteor strike casts though.

**Comment by kaedys** [comment]
> its not worth it to take supremacy usually. i will look into looking for meteor strike casts though.

Supremacy, last I checked, still sims higher than Service, unless the burst from Service is valuable.  In most cases, that burst is valuable, but some fights without significant burst phases (High Command and Felhounds come to mind) can justify Supremacy.

And you're right about Meteor Strike at the start.  I guess that's justifiable.

**Comment by superderpicus** [comment]
yes it sims higher but I believe it’s under the circumstances of lust and CDs on pull. Otherwise using lord of flames will be clunky. it sims higher you’re rught but you don’t really see anyone using it
